### PR TITLE
Increase timeout, fix model argument

### PR DESCRIPTION
**Pull request by liam-sbhoo**
#### Patch
- increase httpx timeout
- fix bug:  `TabPFNClassifier`'s `model` argument didn't work.

*Try to be precise. You can additionally add comments to your PR, this might help the reviewer a lot.*

**If you used new dependencies: Did you add them to `requirements.txt`?**

**Who did you ping on Mattermost to review your PR? Please ping that person again whenever you are ready for another review.**

@SamuelGabriel 

## Breaking changes

If you made any breaking changes, please update the version number.
Breaking changes are totally fine, we just need to make sure to keep the users informed and the server in sync.

**Does this PR break the API? If so, what is the corresponding server commit?**

**Does this PR break the user interface? If so, why?**

---
*Please do not mark comments/conversations as resolved unless you are the assigned reviewer. This helps maintain clarity during the review process.*

### CODE REVIEW

File: tabpfn_client/estimator.py [hunk 1]
```diff
@@ -212,8 +212,8 @@ def fit(self, X, y):
         if config.g_tabpfn_config.use_server:
             try:
                 assert (
-                    self.model == "latest_tabpfn_hosted"
```
Comment (SamuelGabriel) [inline tabpfn_client/estimator.py:215]: Could you please also remove this assert and add the other models, like you did for regression?

**Review by SamuelGabriel — APPROVED** [review]
Could you add the extra models for cls as well pls. That would be nice, even without this is ready to be merged.

File: tabpfn_client/estimator.py (hunk 1 above)
Comment (liam-sbhoo) [reply tabpfn_client/estimator.py:215]: Okay done :)

**Review by SamuelGabriel — APPROVED** [review]
Looks good but for one name :) Thanks for this PR! These are important features :)

File: tabpfn_client/estimator.py [hunk 2]
```diff
@@ -114,7 +114,46 @@ def to_dict(self):
...
+    @classmethod
+    def _validate_model_name(cls, model_name: str) -> None:
+        if model_name != "default" and model_name not in cls._AVAILABLE_MODELS:
+            raise ValueError(
+                f"Invalid model name: {model_name}. "
+                f"Available models are: {cls.list_available_models()}"
+            )
+
+    @classmethod
+    def model_name_to_path(
```
[the remark this answers is not in the document]
Comment (liam-sbhoo) [reply tabpfn_client/estimator.py:137]: Yeah totally, this should stay private.

**Pull request merged**